source/includes/backups/_10-create.md.erb

```diff
@@ -34,39 +52,41 @@ Content-Type: <%= data.apis.versions.backup %>; charset=utf-8
       },
       "find": {
         "href": "https://ci.example.com/go/api/users/:login_name"
+      },
+      "current_user" : {
+        "href" : "https://ci.example.com/go/api/users/current_user"
       }
     },
     "login_name": "username"
   }
 }
 ```
 
-Create a server backup.
-
 <aside class="warning">
   The server may be unavailable during the time that the backup is being taken.
 </aside>
 
-<aside class="notice">
-  The response from the call will not be received till the backup is complete
-  (synchronous). If the data to be backed up is a lot, this can take minutes
-  to complete.
-</aside>
-
-
-<%=
-available_since('15.2.0') do
+<%= available_since('19.3.0') do
```

Comment:
We should add 15.2.0 with the link `https://api.gocd.org/15.2.0/#backups` to the list below for users to refer to v1.

Reply:
Comment incorporated.